scripts/gha/setup-training-operator.sh

```diff
@@ -27,7 +27,7 @@ cd manifests/overlays/standalone
 kustomize edit set image kubeflow/training-operator=${TRAINING_CI_IMAGE}
 
 echo "Installing training operator manifests"
-kustomize build . | kubectl apply -f -
+kustomize build . | kubectl apply --server-side=true -f -
```

Comment:
Do we want to use server side apply to deploy operator ?

Reply:
Client-side apply now fails as the size of the CRDs have increased beyond the (default) maximum "last-applied" annotation size. It's a recurrent issue kubernetes/kubectl#712 that's often faced with CRDs.
Note that server-side apply may eventually become the default for the `kubectl apply` command: kubernetes/enhancements#3805.

Reply:
I see, thanks for sharing